projects/orquestra-sdk/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 🔥 *Features*
 
+* New function: `sdk.ref_infer` to infer git refs for Git Imports
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏I'm not sold on the name, it seems to me that `ref` could be anything and not specific it Git.
Having said that, naming things is hard and what we have is better than delaying evan more so we can keep using it and can always rename later, at the cost of a "deprecation cycle".

Reply:
`sdk.infer_git_ref()`?